automation/taskcluster/lib/tasks.py

```diff
@@ -183,10 +183,10 @@ def craft_test_pr_task(self, variant, run_coverage=False):
                 'tier': 1,
             },
             scopes=[
-                'secrets:get:project/mobile/fenix/pr'
+                'secrets:get:project/mobile/fenix/public-tokens'
```

Comment:
Rather than having different secrets and giving access for each type of build (pr, master, maybe release), I've had some success just having a single TC secret for each type of build and duplicating re-used secrets. This way, we don't need to update `ci-configuration` when a new secret is needed.

If we want code-coverage on both `master` and `pr`, perhaps we should put that secret in `project/mobile/fenix/pr` and `project/mobile/fenix/push`?